WBooks/CommentTableViewController.swift

```diff
@@ -29,13 +29,14 @@ class CommentTableViewController: UITableViewController {
         setTableBackground()
     }
     
-    func setBookID(_ bookID : Int){
-        self.bookID = bookID
+    convenience init(_ bookID : Int){
+        self.init()
+        self._bookID = bookID
         loadComments()
     }
     
     func loadComments(){
-        _commentRepo.fetchComments(bookID!).observe(on: UIScheduler()).startWithResult{
+        _commentRepo.fetchComments(_bookID!).observe(on: UIScheduler()).startWithResult{
```

Comment:
if its an `Int!` you dont need to unwrap it. Also in the constructor you are setting an `Int` so the property doesnt need the `!`